### ISSUE TITLE
Provide button linking to Swapr Pool Info (for Gnosis Markets)

### ISSUE THREAD

**Issue by greenlucid** [seer-pm/demo#142]
<img width="725" alt="image" src="https://github.com/user-attachments/assets/557b7eff-7ac3-4f1d-992d-f8a75bf86f84">

Click it, get taken to swapr / whatever, page providing info on the pools

https://v3.swapr.eth.limo/#/info/pools/0x4a3fec341be7134b8ef9e9edb6bf63ae2ba17f43

This can be important. Swapr interface doesn't make it easy to access this page, but it matters directly to seer markets because liquidity distribution matters to traders. Otherwise, their only way is to try buying/selling shares through Seer UI to guess-work the current liquidity.

Only way to access this pool info from Swapr interface is to click "Pools", wait for it to load 5 pages of pools. If it's there, you'll see it, otherwise it's hidden. **No links from positions to pool info**. So, for non-savvy users, virtually all Pool Info is completely inaccessible.

Sort of info you can get through looking at liquidity distribution:

<img width="1150" alt="image" src="https://github.com/user-attachments/assets/99087c21-f94d-4162-9c77-45518372ae2e">

"Oh, there's enough liquidity for me to sell my 1000 shares!"
"Oh, there's barely any liquidity, there's an oportunity for me to provide the liquidity and get incentives!"

etc. In the absence of an Order Book like Polymarket has, and until that feature is implemented, providing an easy link to this information is the only way to provide this info.

**Comment by greenlucid** [comment]
(a bit of a rant, hopefully doesn't come up as rude)

For all the discussion and thoughts on **Liquidity** that Seer has made, the current interface provides no information about liquidity, only current prices. Liquidity is the arguably the most crucial piece of information for Prediction Markets, you have to be transparent about it and exhaustive. The current "Open Interest" displayed only displays information on how much tokens are currently minted, but that has nothing to do with trading.
Polymarket doesn't display that information. Most users won't understand the point. What's the difference between holding sDAI and the existence of minted outcome tokens that are not making a market, or are not available close to the current spread? There's no difference, so why report it? Those interests are not earned thanks to Seer, so why report them?

Instead, I would just display a number aggregating some relevant liquidity information, such as how much liquidity is close to the spread.

tl;dr pls remove "Open Interest" and switch it for a "Liquidity" metric, because that's what users will expect from that info
